api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/domain/Oppgave.java

```diff
@@ -5,12 +5,14 @@
 public class Oppgave implements Serializable {
 
     public final String oppgaveId, fnr, henvendelseId;
+    public final boolean erSTOOppgave;
     public String svarHenvendelseId;
 
-    public Oppgave(String oppgaveId, String fnr, String henvendelseId) {
+    public Oppgave(String oppgaveId, String fnr, String henvendelseId, boolean erSTOOppgave) {
```

Comment:
Kunne ikke man har lagt et Enum med type STOOppgave  ellers noen type for oppgave som har lagt av veileder ? Ellers er det overkill ?

Reply:
Det er nok mulig, men siden det bare vil være to mulige verdier så tenkte jeg at det ble litt mye. :P